DevelopedCalculator.js

```diff
@@ -9,9 +10,13 @@ class DevelopedCalculator extends Calculator {
         this.hostElem.prepend(historyDiv);
 
 
-        this.updateResultElement();
+        const currentResultUpdate = []
```

Comment:
is this the array that meant to hold the values of the `currentResult`

Reply:
yes

Reply:
will you need to access this array in `onButtonNumberClick_` when a button number clicked?